modules/cudl-data-processing/sns.tf

```diff
@@ -12,7 +12,7 @@ resource "aws_sns_topic" "source_item_updated" {
         "Resource": "arn:aws:sns:*:*:${var.environment}-${each.key}-event-notification-topic",
         "Condition": {
             "ArnLike": {
-                "aws:SourceArn": "${aws_s3_bucket.transform-lambda-source-bucket[each.key].arn}"
+                "aws:SourceArn": "${local.transform-lambda-buckets[each.key].arn}"
```

Comment:
This resource `aws_sns_topic.source_item_updated` now refers to both the source bucket and the dest bucket - would it make sense to rename the resource to reflect the dual purpose?

Reply:
I've changed this from `source_item_updated` to `handle_multiple_processing_event` as these SNS topics are used when we need to process an event multiple ways.